functions-app/config.js

```diff
@@ -31,6 +32,7 @@ module.exports = {
     Port: process.env.SQL_PORT || 1433,
     Timeout: process.env.SQL_TIMEOUT || oneMinuteInMilliseconds,
     requestTimeout: parseInt(process.env.SQL_REQUEST_TIMEOUT, 10) || fiveMinutesInMilliseconds,
+    censusRequestTimeout: parseInt(process.env.SQL_CENSUS_REQUEST_TIMEOUT, 10) || twoHoursInMilliseconds,
```

Comment:
i think you will be pleasantly surprised by the speed of the bulk import, but let's keep this high by default to avoid issue